### PR TITLE
fix: adjust footer and nav

### PR DESCRIPTION
**Pull request by ahmaxed**

### CODE REVIEW

File: assets/css/screen.css [hunk 1]
```diff
@@ -2098,21 +2103,36 @@ pre.runkit-element {
...
     text-decoration: underline;
 }
 
+p.footer-donation {
+    font-weight: 700;
+    font-size: 18px;
+}
+
+.footer-container .col-spacer {
+    margin-top: -3rem;
```
Comment (scissorsneedfoodtoo) [inline assets/css/screen.css]: Not sure if this is just on my end, but this seems to cause some overlap in the two "Trending Guides" columns on mobile: 

image

Seems like `margin-top: -1.45rem` could be moved from the media query below to here for the mobile view.

**Comment by scissorsneedfoodtoo** [comment]
Tested on news.dev and everything LGTM. We just need to tweak the CSS slightly for mobile.

File: assets/css/screen.css (hunk 1 above)
Comment (ahmaxed) [reply assets/css/screen.css]: thanks, this has been resolved.

**Comment by ahmaxed** [comment]
@scissorsneedfoodtoo  we can add the magnifying class in a separate pr

**Review by scissorsneedfoodtoo — APPROVED** [review]

**Pull request merged**